test/e2e/cluster_samples_operator_test.go

```diff
@@ -974,6 +976,11 @@ func TestUpgrade(t *testing.T) {
 	newVersion := kapis.Now().String()
 	t.Logf("current version %s version for upgrade %s", cfg.Status.Version, newVersion)
 
+	err = framework.DisableCVOForOperator(operatorClient)
+	if err != nil {
+		t.Fatalf("problem disabling operator deployment in CVO %#v", err)
+	}
```

Comment:
any reason not to do this up front in your main?  Especially since you don't re-enable it, it's odd to do it in the middle of one test.

Reply:
will do